### PR TITLE
cmd/snap: add 'snap skills'

### PR DESCRIPTION
**Pull request by zyga**
This adds the skills command (not 100% to spec but with a TODO to implement the rest). It depends on #457 for some common code.

### CODE REVIEW

File: cmd/snap/cmd_skills.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+	defer w.Flush()
+	for _, skill := range skills {
+		// TODO: support filtering by snap:skill
+		if x.Type != "" && skill.Type != x.Type {
+			continue
+		}
+		fmt.Fprintf(w, "%s:%s\t", skill.Snap, skill.Name)
+		for i := 0; i < len(skill.GrantedTo); i++ {
+			if i > 0 {
+				fmt.Fprintf(w, ",")
```
Comment (chipaca) [inline cmd/snap/cmd_skills.go]: maybe use just `fmt.Fprint` for these?

Comment (zyga) [inline cmd/snap/cmd_skills.go]: Will do, thanks

File: cmd/snap/cmd_skills_test.go [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+				},
+			},
+		})
+	})
+	rest, err := Parser().ParseArgs([]string{"skills"})
+	c.Assert(err, IsNil)
+	c.Assert(rest, DeepEquals, []string{})
+	expectedStdout := "" +
+		"Skill                Granted To\n" +
+		"canonical-pi2:pin-13 \n"
```
Comment (chipaca) [inline cmd/snap/cmd_skills_test.go]: could we write `--` instead of just nothing, when it isn't granted to anything?

**Comment by chipaca** [comment]
:+1:

File: cmd/snap/cmd_skills_test.go (hunk 2 above)
Comment (zyga) [inline cmd/snap/cmd_skills_test.go]: I did that before but this is divergent from the skills spec. I'll discuss this when Gustavo is back.

**Pull request merged**